oss-cloud-backend/handler.js

```diff
@@ -95,16 +95,21 @@ module.exports.addContributor = async (event) => {
 module.exports.updateNextContributor = async () => {
   try {
     const nextToUpdate = await databaseService.nextContributor();
+    console.log('Updating contributor', nextToUpdate.username);
     const results = await gitHubApiService.getContributorPullRequests(nextToUpdate.username);
+    console.log('Updated contributor', nextToUpdate.username);
```

Comment:
Please remove log statements. We can leave some, but don't need all these probably.